examples/SharedMemory/PhysicsServerCommandProcessor.cpp

```diff
@@ -9133,6 +9247,7 @@ bool PhysicsServerCommandProcessor::processSendPhysicsParametersCommand(const st
 				}
 			};
 
+#ifdef SKIP_DEFORMABLE_BODY                    
```

Comment:
functionality is removed, such as switching constraint solver. It is probably better to leave the default world/solver and add an API to enable deformable.

Reply:
I changed the default back to softMultiBodyWorld.